sdks/java/core/src/main/java/org/apache/beam/sdk/values/PValue.java

```diff
@@ -22,6 +22,8 @@
 
 /**
  * The interface for values that can be input to and output from {@link PTransform PTransforms}.
+ *
+ * <p>It is recommended to extend {@link PValueBase}
```

Comment:
Ideally `PCollectionView` will not be a `PValue` by the stable release, and this interface can also disappear.